src/utils/routeUtils.ts

```diff
@@ -0,0 +1,10 @@
+export const RouteFormat = (
```

Comment:
```suggestion
export const getFormattedRoute = (
```
Use proper naming conventions for functions.